Update Post “2018-10-gnomad-v2-1” (#67)
broadinstitute/gnomad-blog: content/posts/2018-10-gnomad-v2-1.md

- `freq`: Allele frequency information (AC, AN, AF, homozygote count)
- `age_hist_het`: Histogram of heterozygous carriers
- `age_hist_hom`: Histogram of homozygous carriers
- `popmax`: Allele frequency information for the non-bottlenecked population with the highest frequency.

>   This excludes Finns, Ashkenazi Jewish and “Other” populations.

Reviewer: ```suggestion
  This excludes Ashkenazi Jewish (asj), European Finnish (fin), and "Other" (oth) populations.
```

And finally, special thanks to the Broad Institute Genomics Platform as a whole, who generated 90%
of the data used in gnomAD, and also provided critical computing resources required for this work.

> Post has been edited on April 20, 2022 to rephrase "outbred population" as "non-bottlenecked population".

Reviewer: ```suggestion
*Post was updated on April 20, 2022 to rephrase "outbred population" as "non-bottlenecked population"*.
```
